yew/src/virtual_dom/mod.rs

```diff
@@ -390,12 +396,17 @@ impl ToString for Classes {
     }
 }
 
+impl From<Cow<'static, str>> for Classes {
+    fn from(t: Cow<'static, str>) -> Self {
+        let mut set = IndexSet::new();
+        set.insert(t);
```

Comment:
There was a reason I went with delegating the work to `Self::from` instead of just creating a new set with a single element: We don't know whether the value contains multiple classes!

Reply:
I forgot about it :sweat:

Reply:
Fixed